test/e2e/compact_test.go

```diff
@@ -509,9 +509,6 @@ func TestCompactWithStoreGateway(t *testing.T) {
 		testutil.Ok(t, c.WaitSumMetrics(e2e.Equals(3), "thanos_compact_group_compaction_runs_started_total"))
 		testutil.Ok(t, c.WaitSumMetrics(e2e.Equals(2), "thanos_compact_group_compaction_runs_completed_total"))
 
-		testutil.Ok(t, c.WaitSumMetrics(e2e.Equals(0), "thanos_compact_downsample_total"))
```

Comment:
Why this was removed. (:

Reply:
This metric and `thanos_compact_downsample_failures_total` are both missing. They are not even initialized to 0 because in this test case, the compactor halts so it doesn't actually run the downsample logic @bwplotka

Reply:
well that's the key here (: We don't expect so we set expectancy to 0

Reply:
but whatever, we can think about it later